doc/classes/Window.xml

```diff
@@ -11,6 +13,7 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether the window is being drawn to the screen.
```

Comment:
Is this correct? `can_draw` implies that it is _able_ to do something, e.g. `if (can_draw()) { draw(); }`. If this description is correct, shouldn't the method be called something like `is_drawn`/`is_drawing`

Reply:
I've tested all of the logic to these descriptions that I've added, and I can confirm that this method will return `true` if the Window is visible, but `false` if it's not (i.e "is it being drawn to the screen right now?"). As far as I can tell it's almost identical to the `Window.visible` property in almost every respect.

Not really sure why it's here honestly, it seems redundant in most cases. Unless there's some really specific or obscure use for this function, that I'm not thinking of. I agree that this method should probably be called `is_drawn` instead, as that's a lot less misleading.

<img width="946" alt="can_draw" src="https://user-images.githubusercontent.com/62965063/124699689-44d41d80-deb9-11eb-983f-91dba8b5ad66.PNG">

<img width="713" alt="can't_draw" src="https://user-images.githubusercontent.com/62965063/124699704-4d2c5880-deb9-11eb-98d4-91afd0b9dc56.PNG">